packages/serverless/src/apollo/app/collection.ts

```diff
@@ -54,7 +54,7 @@ export const CollectionCreateRequest = Joi.object<TCollectionCreateRequest>({
   groupName,
   indexes: Joi.array().items(collectionIndex.optional()),
   schema: schemaFields,
-  permissions: permissionDetail,
+  permission: permissionDetail,
```

Comment:
It may not `permissionDetail`, it suppose to be `Joi.number().optional()`

Reply:
Solved